node-registrar/pkg/db/farms.go

```diff
@@ -49,10 +49,17 @@ func (db *Database) CreateFarm(farm Farm) (uint64, error) {
 	return farm.FarmID, nil
 }
 
-func (db *Database) UpdateFarm(farmID uint64, name string) (err error) {
-	result := db.gormDB.Model(&Farm{}).Where("farm_id = ?", farmID).Updates(map[string]interface{}{
-		"farm_name": name,
-	})
+func (db *Database) UpdateFarm(farmID uint64, name string, stellarAddr string) (err error) {
+	update := map[string]interface{}{}
+
+	if len(name) != 0 {
+		update["farm_name"] = name
+	}
+	if len(stellarAddr) != 0 {
```

Comment:
assert that it's 56, and begins with G i think

Reply:
it's validated here https://github.com/threefoldtech/tfgrid4-sdk-go/pull/33/files#diff-9c906af56e749b0eab37fe8d78e813ac3945dfb10d1c335c5cc24f81a2a32aa1R134

Reply:
Isn't that working only for update request only?

Reply:
added validation to the farm model too, rules will be applied on creation/update